packages/mint-components/src/components/sqm-brand/sqm-brand.tsx

```diff
@@ -0,0 +1,84 @@
+import { withHooks } from "@saasquatch/stencil-hooks";
+import { useEffect } from "@saasquatch/universal-hooks";
+import { Component, h, Host, Prop } from "@stencil/core";
+import { buildFontsCssUrl } from "../../fonts/GoogleFonts";
+import { autoColorScaleCss } from "../sqm-stencilbook/AutoColor";
+
+/**
+ *
+ * @uiName Brand Container
+ * @slot Controls the brand color and font of Mint Components wrapped by the container.
+ */
+@Component({
+  tag: "sqm-brand",
+  shadow: true,
+})
+export class BrandComponent {
+  /**
+   * Controls the primary brand color used in the Mint Components library. Note that this
+   * does not affect vanilla components or other component libraries.
+   *
+   * @uiName Brand Color
+   * @uiWidget color
+   */
+  @Prop() brandColor: string;
+
+  /**
+   * The brand font that you want to use
+   *
+   * @uiName Brand Font
+   */
+  @Prop() brandFont: string;
```

Comment:
We might actually want to apply a default here, default being Nunito Sans - as technically we would overwrite the default font with Arial given the current code on lines 70-73

Reply:
Or apply a similar if state like we did for brandColour on line 75

Reply:
Added `Nunito Sans` as default

Reply:
Why not a controller level default instead of just in the render function?